README.md

```diff
@@ -1,7 +1,20 @@
 # docs-vim
-## Cloning Instructions
-This repository uses ``git submodule``, after cloning it you should run ``git
-submodule init`` and ``git submodule update``.
+## Installation
```

Comment:
Can we rename this to "Installation (Chrome)"? Just in case we want to add Firefox instructions sometime :)

Reply:
I edited it.